cmd/gcp-controller-manager/csr_signer_test.go

```diff
@@ -136,23 +136,9 @@ func TestGKESigner(t *testing.T) {
 		},
 		{
 			name: "Signs kubelet serving certificates",
-			csr: &certsv1b1.CertificateSigningRequest{
+			csr: &certsv1.CertificateSigningRequest{
 				Spec: capi.CertificateSigningRequestSpec{
-					SignerName: stringPointer(certsv1b1.KubeletServingSignerName),
-					Request:    generateCSR(),
-				},
-				Status: statusApproved,
-			},
-			mockResponse:  goodResponse,
-			expected:      goodResponse.Status.Certificate,
-			wantProcessed: true,
-			wantErr:       false,
-		},
-		{
-			name: "Signs legacy-unknown certificates",
-			csr: &certsv1b1.CertificateSigningRequest{
-				Spec: capi.CertificateSigningRequestSpec{
-					SignerName: stringPointer(certsv1b1.LegacyUnknownSignerName),
```

Comment:
restore this test case, legacy signer CSRs could still be created via the v1beta1 API and observed by this controller via the v1 API

Reply:
Done